### ISSUE TITLE
SonarQube 8.9.0: Cannot be activated on jsp profile FindBugs Security JSP

### ISSUE THREAD

**Issue by alexei-28** [spotbugs/sonar-findbugs#389]
CentOS 7, 
SonarQube 8.9.0,
java 11

Install success and run. Nice.

But when I install pluggin (Administration->Marketplace) **Java FindBugs** then SonarQube 8.9.0 is not started. 

Here error:

```
2021.09.15 15:45:04 ERROR web[][o.s.s.p.Platform] Background initialization failed. Stopping SonarQube
org.sonar.server.exceptions.BadRequestException: java rule findsecbugs:XSS_JSP_PRINT cannot be activated on jsp profile FindBugs Security JSP
    at org.sonar.server.exceptions.BadRequestException.create(BadRequestException.java:57)
    at org.sonar.server.exceptions.BadRequestException.create(BadRequestException.java:61)
    at org.sonar.server.exceptions.BadRequestException.checkRequest(BadRequestException.java:44)
    at org.sonar.server.qualityprofile.RuleActivationContext.doSwitch(RuleActivationContext.java:227)
    at org.sonar.server.qualityprofile.RuleActivationContext.reset(RuleActivationContext.java:208)
    at org.sonar.server.qualityprofile.RuleActivator.activate(RuleActivator.java:76)
    at org.sonar.server.qualityprofile.BuiltInQProfileUpdateImpl.update(BuiltInQProfileUpdateImpl.java:73)
    at org.sonar.server.qualityprofile.RegisterQualityProfiles.update(RegisterQualityProfiles.java:132)
    at org.sonar.server.qualityprofile.RegisterQualityProfiles.lambda$start$1(RegisterQualityProfiles.java:93)
    at com.google.common.collect.ImmutableList.forEach(ImmutableList.java:405)
    at org.sonar.server.qualityprofile.RegisterQualityProfiles.start(RegisterQualityProfiles.java:88)
    at org.sonar.core.platform.StartableCloseableSafeLifecyleStrategy.start(StartableCloseableSafeLifecyleStrategy.java:40)
    at org.picocontainer.injectors.AbstractInjectionFactory$LifecycleAdapter.start(AbstractInjectionFactory.java:84)
    at org.picocontainer.behaviors.AbstractBehavior.start(AbstractBehavior.java:169)
    at org.picocontainer.behaviors.Stored$RealComponentLifecycle.start(Stored.java:132)
    at org.picocontainer.behaviors.Stored.start(Stored.java:110)
    at org.picocontainer.DefaultPicoContainer.potentiallyStartAdapter(DefaultPicoContainer.java:1016)
    at org.picocontainer.DefaultPicoContainer.startAdapters(DefaultPicoContainer.java:1009)
    at org.picocontainer.DefaultPicoContainer.start(DefaultPicoContainer.java:767)
    at org.sonar.core.platform.ComponentContainer.startComponents(ComponentContainer.java:136)
    at org.sonar.server.platform.platformlevel.PlatformLevel.start(PlatformLevel.java:90)
    at org.sonar.server.platform.platformlevel.PlatformLevelStartup.access$001(PlatformLevelStartup.java:49)
    at org.sonar.server.platform.platformlevel.PlatformLevelStartup$1.doPrivileged(PlatformLevelStartup.java:87)
    at org.sonar.server.user.DoPrivileged.execute(DoPrivileged.java:45)
    at org.sonar.server.platform.platformlevel.PlatformLevelStartup.start(PlatformLevelStartup.java:84)
    at org.sonar.server.platform.PlatformImpl.executeStartupTasks(PlatformImpl.java:198)
    at org.sonar.server.platform.PlatformImpl.access$400(PlatformImpl.java:46)
    at org.sonar.server.platform.PlatformImpl$1.lambda$doRun$1(PlatformImpl.java:122)
    at org.sonar.server.platform.PlatformImpl$AutoStarterRunnable.runIfNotAborted(PlatformImpl.java:370)
    at org.sonar.server.platform.PlatformImpl$1.doRun(PlatformImpl.java:122)
    at org.sonar.server.platform.PlatformImpl$AutoStarterRunnable.run(PlatformImpl.java:354)
    at java.base/java.lang.Thread.run(Thread.java:829)
2021.09.15 15:45:04 INFO  web[][o.s.p.ProcessEntryPoint] Hard stopping process
2021.09.15 15:45:04 INFO  web[][o.s.s.n.NotificationDaemon] Notification service stopped

```

**Comment by gtoison** [comment]
Hello, we currently have an issue that seems to be linked to [SONAR-15240](https://jira.sonarsource.com/browse/SONAR-15240).
Are you using a SonarQube installation that was progressively upgraded to 8.9?

We are discussing the problem and a fix [here](https://github.com/spotbugs/sonar-findbugs/issues/387)
In the meantime reverting to version 4.0.3 of the plugin should work.

**Comment by alexei-28** [comment]
> Hello, we currently have an issue that seems to be linked to [SONAR-15240](https://jira.sonarsource.com/browse/SONAR-15240).
> Are you using a SonarQube installation that was progressively upgraded to 8.9?
> 
> We are discussing the problem and a fix [here](https://github.com/spotbugs/sonar-findbugs/issues/387)
> In the meantime reverting to version 4.0.3 of the plugin should work.

Yes, Before I has **SonarQube 4.9** and progressively upgrade to **8.9**

**Comment by derhecht** [comment]
Same problem in SonarQube 8.9.2 running in Docker. Just to try update from 4.0.3 to 4.0.4 - SonarQube could not start with mentioned error message.

**Comment by gtoison** [comment]
Please follow https://github.com/spotbugs/sonar-findbugs/issues/387 since it is the same issue

**Comment by gtoison** [comment]
We have released version 4.0.5 with a fix for this bug: https://github.com/spotbugs/sonar-findbugs/releases/tag/4.0.5
It was submitted to the marketplace but not accepted yet (there seem to be a backlog for accepting new releases of plugins).
